configtx/orderer.go

```diff
@@ -1060,3 +1105,49 @@ func blockDataHashingStructureValue() *standardConfigValue {
 		},
 	}
 }
+
+func encodeBFTBlockVerificationPolicy(consenterProtos []common.Consenter, ordererGroup *cb.ConfigGroup) error {
```

Comment:
This copied from Fabric right? 
I think it would have been preferable to import it from Fabric, however, unfortunately, Fabric itself uses fabric-config in the integration tests.

Reply:
Yes, then how do you prefer to resolve it?

Reply:
any news @tock-ibm ?